src/main/java/com/sharp/vn/its/management/entity/SystemEntity.java

```diff
@@ -16,26 +16,32 @@
 @Getter
 @Setter
 @Table(name = "its_system")
-public class SystemEntity extends BaseEntity{
+public class SystemEntity extends BaseEntity {
```

Comment:
Why has the format of this file changed

Reply:
The format of this file changed because i used shortcut ctrl+shift+L to format code